src/main/io/serial_4way.c

```diff
@@ -76,9 +79,9 @@
 // *** change to adapt Revision
 #define SERIAL_4WAY_VER_MAIN 20
 #define SERIAL_4WAY_VER_SUB_1 (uint8_t) 0
-#define SERIAL_4WAY_VER_SUB_2 (uint8_t) 03
+#define SERIAL_4WAY_VER_SUB_2 (uint8_t) 04
```

Comment:
Not sure how this is related to the bidirectional Dshot changes. Should this go into a separate pull request?

Reply:
This is required for the extension to the 4way esc protocol which allows resetting the esc. I need to reset the esc to install jesc so it is related to dshot bidirectional support on blhelis hw. I could still move it into a different PR if you think that would be beneficial.

Reply:
Yes please - this could potentially be contentious if it turns out to create issues with other hardware using the 4way protocol, so having it isolated will make resolving this easier.